server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -68,7 +69,8 @@
     private long seqNo;
     private long primaryTerm;
     private boolean exists;
-    private Map<String, DocumentField> fields;
+    private Map<String, DocumentField> otherFields;
```

Comment:
I think keeping "fields", or calling this "documentFields" would be good. "other" fields sounds like this are something extraneous, when in fact they are the main fields.

Reply:
renamed for `documentFields`